Mathlib/Order/Filter/Basic.lean

```diff
@@ -3325,3 +3325,23 @@ theorem Set.MapsTo.tendsto {α β} {s : Set α} {t : Set β} {f : α → β} (h
     Filter.Tendsto f (𝓟 s) (𝓟 t) :=
   Filter.tendsto_principal_principal.2 h
 #align set.maps_to.tendsto Set.MapsTo.tendsto
+
+/-- Construct a filter from a property that is stable under finite intersections.
+This constructor deduces `Filter.univ_sets` and `Filter.inter_sets` from the finite
+intersection property. -/
+def Filter.ofProp_finite_inter {p : Set α → Prop} (he : p univ) (hmono : ∀ ⦃s t⦄, s ⊆ t → p s → p t)
+    (hinter : ∀ ⦃s t⦄, p s → p t → p (s ∩ t)) : Filter α where
```

Comment:
These assumptions are definitionally equal to the fields of `Filter`. I don't think that we need a separate constructor for that.

Reply:
Okay, I'll remove this one!